backend/src/api/model/realm/mutations.rs

```diff
@@ -410,3 +421,10 @@ pub(crate) struct RemovedRealm {
 pub struct CreateRealmLineageOutcome {
     pub num_created: i32,
 }
+
+#[derive(juniper::GraphQLObject)]
+#[graphql(Context = Context)]
+pub(crate) struct RemoveMountedSeriesOutcome {
+    pub removed_realm: Option<RemovedRealm>,
+    pub removed_block: Option<RemovedBlock>,
+}
```

Comment:
It seems like there is always exactly one of these fields set to `Some`. Rather sounds like an `enum` (Rust) /  `union` (graphql) to me?